Parsers/ASimRegistryEvent/test/VMwareCarbonBlackCloud_vimRegistryEvent_SchemaTest.csv

```diff
@@ -0,0 +1,29 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [RegistryPreviousKey]"
+"(1) Warning: Missing recommended field [RegistryPreviousValueData]"
+"(1) Warning: Missing recommended field [RegistryPreviousValueType]"
+"(1) Warning: Missing recommended field [RegistryPreviousValue]"
+"(1) Warning: Missing recommended field [RegistryValueData]"
+"(1) Warning: Missing recommended field [RegistryValueType]"
+"(1) Warning: Missing recommended field [RegistryValue]"
+"(2) Info: Missing optional field [ActingProcessGuid]"
+"(2) Info: Missing optional field [ActorSessionId]"
+"(2) Info: Missing optional field [DstDescription]"
+"(2) Info: Missing optional field [DvcDescription]"
+"(2) Info: Missing optional field [DvcMacAddr]"
+"(2) Info: Missing optional field [DvcOsVersion]"
+"(2) Info: Missing optional field [DvcScopeId]"
+"(2) Info: Missing optional field [EventOriginalSubType]"
+"(2) Info: Missing optional field [EventProductVersion]"
+"(2) Info: Missing optional field [EventReportUrl]"
+"(2) Info: Missing optional field [ParentProcessGuid]"
+"(2) Info: Missing optional field [SrcDescription]"
+"(2) Info: extra unnormalized column [ActorUserType]"
+"(2) Info: extra unnormalized column [DvcAction]"
+"(2) Info: extra unnormalized column [DvcDomainType]"
+"(2) Info: extra unnormalized column [DvcDomain]"
+"(2) Info: extra unnormalized column [DvcFQDN]"
+"(2) Info: extra unnormalized column [DvcIdType]"
+"(2) Info: extra unnormalized column [EventSchema]"
+"(2) Info: extra unnormalized column [EventVendor]"
```

Comment:
seems like we are missing this fields in tester. Can you pls help adding them?

Reply:
Done